docker/multi-process/scripts/init

```diff
@@ -5,6 +5,15 @@ export LC_ALL=en_US.UTF-8
 
 cd /app
 
+# Do we have a Rails secret yet?
+# If not, generate one, but persist it for the container's lifetime.
+if grep -q ^APP_SECRET_TOKEN=REPLACE_ME_NOW .env.example; then
+  echo 'Generating random APP_SECRET_TOKEN.'
+  secret=$(dd if=/dev/urandom bs=36 count=1 | openssl base64)
+  sed "s:REPLACE_ME_NOW:$secret:" .env.example >.env.new
```

Comment:
Yeah, this is brain habit -- `sed -i` is a GNUism that I always forget. Will do.